Audits Bridges (#1204)
maticnetwork/matic-docs: docs/develop/plasma-using-metamask.md

## Example using Plasma ERC20

:::note

> For the mainnet, Ethereum is the root chain and Polygon Mainnet is the child chain and for the testnet, Goerli is the root chain and Mumbai is the child chain. The values in config.json file should be set accordingly. Goerli and Mumbai networks are used as the root and child chain in this tutorial.

Reviewer: ```suggestion
For the mainnet, Ethereum is the root chain and Polygon Mainnet is the child chain and for the testnet, Goerli is the root chain and Mumbai is the child chain. The values in the `config.json` file should be set accordingly. Goerli and Mumbai networks are used as the root and child chain in this tutorial.
```

`parentProvider: window.web3`

:::note 

> Deposits from Ethereum to Polygon happen using a state sync mechanism and takes about 22-30 minutes After waiting for this time interval, it is recommended to check the balance using web3.js/matic.js library or using Metamask. The explorer will show the balance only if at least one asset transfer has happened on the child chain. This [link](/docs/develop/ethereum-polygon/plasma/deposit-withdraw-event-plasma/) explains how to track the deposit events.

Reviewer: ```suggestion
Deposits from Ethereum to Polygon happen using a state sync mechanism and take about 22-30 minutes. After waiting for this time interval, it is recommended to check the balance using the web3.js/matic.js library or using Metamask. The explorer will show the balance only if at least one asset transfer has happened on the child chain. This [link](/docs/develop/ethereum-polygon/plasma/deposit-withdraw-event-plasma/) explains how to track the deposit events.
```

const txReceipt = await result.getReceipt();
```

> MATIC is native token on Polygon. So we support transfer of Matic tokens without any token address.

Reviewer: ```suggestion
MATIC is the native token of the Polygon network. So we support the transfer of MATIC tokens without any token address.
```
